examples/neuralNetwork-sequence-hand-gesture-load-model/sketch.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * 👋 Hello! This is an ml5.js example made and shared with ❤️.
+ * Learn more about the ml5.js project: https://ml5js.org/
+ * ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ *
+ * This example demonstrates loading a hand gesture classifier through
+ * ml5.neuralNetwork with the sequenceClassificationWithCNN task.
+ * This example has been trained with the ASL gestures for Hello and Goodbye.
+ *
+ * Reference to sign hello and goodbye in ASL:
+ * Hello: https://www.signasl.org/sign/hello
+ * Goodbye: https://www.signasl.org/sign/goodbye
+ */
+
+let video;
+let handPose;
+let hands = [];
+let model;
+let isModelLoaded = false;
+
+let sequence = [];
+let sequenceLength = 50;
```

Comment:
@mop9047 Change this to 30? If you can, please also upload re-trained model files (with 84 dimensions)

Reply:
That's great - thank you!

I think both (showing or not showing the path in this example) could work. Perhaps leaning a bit towards the latter, so that there is more differentiation to what your other hand-gesture example offers, and we keep this a way of demonstrating how a clean, "user-facing" application could look like?